packages/ember-htmlbars/tests/attr_nodes/style_test.js

```diff
@@ -71,3 +71,24 @@ QUnit.test('specifying `<div style={{userValue}}></div>` works properly with a S
 
   deepEqual(warnings, [ ]);
 });
+
+QUnit.test('null value do not generate htmlsafe warning', function() {
+  view = EmberView.create({
+    userValue: null,
+    template: compile('<div style={{userValue}}></div>')
```

Comment:
After re-reviewing, I believe that this (and the other test) need to be tweaked to either use {{view.userValue}} (as it stands this one that is testing undefined also because there is no context of this view).